data/avro-schema/src/main/resources/avro/net/corda/data/ledger/persistence/TransactionOutput.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "TransactionOutput",
+  "doc": "Transaction output data of signed transaction.",
+  "namespace": "net.corda.data.ledger.persistence",
+  "fields": [
+    {
+        "name": "id",
+        "type": "string",
```

Comment:
Would not be net.corda.data.crypto.SecureHash better for this?

Reply:
I've kept it consistent with types we used so far (e.g. as in https://github.com/corda/corda-api/blob/release/os/5.0/data/avro-schema/src/main/resources/avro/net/corda/data/ledger/persistence/FindTransaction.avsc)